src/zzz_od/api/routers/onedragon.py

```
@@ -54,19 +82,123 @@ def _exec():
+    import requests
+    response = requests.get("http://localhost:8000/api/v1/onedragon/auto-battle-configs")
+    configs = response.json()
+    for config in configs:
+        print(f"自动战斗配置: {config}")
+    ```
+    """
+    try:
+        config_list = get_auto_battle_op_config_list('auto_battle_state_handler')
+        # 提取配置名称
+        config_names = [config.value for config in config_list]
+        return config_names
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)